Formula/nushell.rb

```diff
@@ -19,8 +18,8 @@ class Nushell < Formula
   # Nu requires features from Rust 1.39 to build, so we can't use Homebrew's
   # Rust; picking a known-good Rust nightly release to use instead.
   resource "rust-nightly" do
-    url "https://static.rust-lang.org/dist/2019-08-24/rust-nightly-x86_64-apple-darwin.tar.xz"
-    sha256 "104ddea51b758f4962960097e9e0f3cabf2c671ec3148bc745344431bb93605d"
+    url "https://static.rust-lang.org/dist/2019-10-13/rust-nightly-x86_64-apple-darwin.tar.xz"
```

Comment:
Is it still true that it requires `rust-nightly`? That should not be acceptable in Homebrew.

Reply:
Yeah, I assume it might still be true.

There is another formula also requires the `rust-nightly` as well.

Reply:
[It requires beta now.](http://www.jonathanturner.org/2019/10/nushell-0_4_0.html#onto-the-beta-compiler-est31)

Reply:
That is good to know. 😆

Reply:
We shouldn't bump this either URL or formula until it works with stable `rust`.

Reply:
Updating to the new version makes homebrew strictly better ... especially since it's already in.

Otherwise please remove the formula; I would have rather not wasted time with the old version from homebrew.

Reply:
> Otherwise please remove the formula

No, sorry.